### DISCUSSION TITLE
Logeo en Colab

### DISCUSSION THREAD

**Discussion by juliancorredor99** [rramosp/ai4eng.v1#131] [01 - INTRODUCCIÓN]
Hola, lo que pasa es que no sé porque me aparecen dos cursos, uno se llama 20221 y el otro 2022.1, entonces cuando me quiero logear en el colab para hacer los labs no me deja por eso mismo.

**Comment by Felipe-RA** [comment]
Saludos julian

Mirate el post  enlazado en la plataforma y sigue las instrucciones dejando tu correo para que solucionemos

**Comment by sofiagutz** [comment]
Buenas Felipe, 
Espero que se encuentre bien, le escribo porque estaba realizando los laboratorios del módulo 1 y 2, pero tengo un problema al intentar loguearme para poder validar mis actividades realizadas. No tengo el curso duplicado, entonces no entiendo qué puede suceder. 

Además de eso, no sé si ustedes este semestre volvieron a compartir las carpetas de drive para cada persona, pues yo no tengo una, y antes de guardar las copias del laboratorio me gustaría saber si debo guardarlas en un lugar en específico. 
Quedo atenta y mil gracias.

**Reply by Felipe-RA** [reply]
Saludos sofia

Necesito más detalles del problema para poderte ayudar

Este semestre no hay carpetas de drive por cambios en las cuentas institucionales, te recomiendo te mires las grabaciones de las dos primeras sesiones y asistas mañana a la sesión de github

**Reply by sofiagutz** [reply]
![1](https://user-images.githubusercontent.com/83871876/173864555-f001da01-2575-4888-b93c-a8e2fb7de588.png)
![2](https://user-images.githubusercontent.com/83871876/173864562-fe4bd2f3-4e5a-4555-92d5-ce6fb8443239.png)
cuando intento loguearme tanto manualmente como por google me aparece esto. Necesito solucionar este problema hoy para poder avanzar  rápido

**Reply by Felipe-RA** [reply]
Como dice el mensaje de error, el problema está en tu usuario / contraseña.

Revisa que si te hayas inscrito con el correo correcto

También cambia tu contraseña desde rlxmooc

**Comment by rramosp** [comment]
sofia, tienes que usar el login/pwd ... con google auth no funciona, ya que
google hizo unos cambios en su api que incompatibilizan esto

On Wed, Jun 15, 2022 at 10:21 AM Sofía Gutiérrez T. <
***@***.***> wrote:

> [image: 1]
> <https://user-images.githubusercontent.com/83871876/173864555-f001da01-2575-4888-b93c-a8e2fb7de588.png>
> [image: 2]
> <https://user-images.githubusercontent.com/83871876/173864562-fe4bd2f3-4e5a-4555-92d5-ce6fb8443239.png>
> cuando intento loguearme tanto manualmente como por google me aparece
> esto. Necesito solucionar este problema hoy para poder avanzar rápido
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/131#discussioncomment-2956972>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJCMO3MJ4LL5UUNVXDVPHYJDANCNFSM5XSWCY3A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
